### PR TITLE
feat(icon-circle): Create component that represents icons in circles.

### PR DESCRIPTION
**Pull request by Keikonichi**
Create component that represents icons in circles.

### CODE REVIEW

File: app/carreira/page.tsx [hunk 1]
```diff
@@ -1,11 +1,60 @@
+import IconText from '../components/IconText';
```
Comment (renatoagomes) [inline app/carreira/page.tsx]: avoid relative paths. use '@/components/...' instead

File: app/quemsomos/page.tsx [hunk 2]
```diff
@@ -1,12 +1,60 @@
 import CompanyCulture from '../components/CompanyCulture';
+import IconText from '../components/IconText';
 import TeamMemberList from '../components/TeamMemberList';
 import TextWithButton from '../components/TextWithButton';
```
Comment (renatoagomes) [inline app/quemsomos/page.tsx]: same from above, use the alias for the root path '@/...'

**Comment by Keikonichi** [comment]
In screen 1920 px the page career

image


in screen 1024 px the page career

image

in screen 425 px the page career

image

**Comment by Keikonichi** [comment]
In the screen 1920 px the About us page.

In the screen 1920 px the About us page.

**Comment by Keikonichi** [comment]
In the screen 1920 px the About us page.

image

In the screen 1024 px the About us page.

image

In the screen 425 px the Abut us page.

image

**Comment by Keikonichi** [comment]
Finished the task 

@renatoagomes 
@thiagobraga

**Comment by thiagobraga** [comment]
Very nice @Keikonichi congratulations :blush:
I think we can accept the MR. @renatoagomes

**Pull request merged**